lib/open_project/translations/models/locales_updater_configuration.rb

```diff
@@ -0,0 +1,14 @@
+require 'yaml'
+
+class LocalesUpdaterConfiguration
+  CONFIGURATION_FILE = ENV['CONFIGURATION_FILE'] || begin
```

Comment:
I wonder again why this is a constant - is it intended to be accessible from outside?

That would read something like

```ruby 
LocalesUpdaterConfiguration::CONFIGURATION_FILE
```

and that sounds a bit awkward. If it should be accessible from the outside, I'd suggest the name `FILE`..

I'd recommend putting this constant into a method (see above), as you could probably catch any errors on call.